packages/launcher/static/applications/onyx-stats/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "node": "current"
```

Comment:
For now it's probably safer to put the chrome or a generic "browsers" target, as the supported ES features might be different with node.